src/test/java/core/basesyntax/strategy/BalanceOperationTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationTest {
+    private BalanceOperation balanceOperation;
+    private Map<String, Integer> inventory;
+
+    @BeforeEach
+    public void setUp() {
+        balanceOperation = new BalanceOperation();
+        inventory = new HashMap<>();
+    }
```

Comment:
Same here and in other `Operation` tests, you should do it like this for example:

```java
@BeforeAll
    static void beforeAll() {
        inventory = new HashMap<>();
        balanceOperation = new BalanceOperation();
    }
```